scripts/spack/devtools_configs/toss_4_x86_64_ib/spack.yaml

```diff
@@ -138,6 +144,16 @@ spack:
       externals:
       - spec: findutils@4.6.0
         prefix: /usr
+    gcc:
+      externals:
+      - spec: gcc@10.3.1 languages='c,c++,fortran'
+        prefix: /usr/tce/packages/gcc/gcc-10.3.1
+        extra_attributes:
+          compilers:
+            c: /usr/tce/packages/gcc/gcc-10.3.1/bin/gcc
+            cxx: /usr/tce/packages/gcc/gcc-10.3.1/bin/g++
+            fortran: /usr/tce/packages/gcc/gcc-10.3.1/bin/gfortran
+      buildable: false
```

Comment:
were the devtools built with pre-1.0 spack?

Reply:
I built them with Spack 1.0 then ran into some bugs.. worked through most of them and gave up and reverted them to the previous devtools.